### ISSUE TITLE
cli vignettes could now use downlit

### ISSUE THREAD

**Issue by hadley** [r-lib/cli#263]
Instead of ansiicast.

**Comment by gaborcsardi** [comment]
That would be great, to avoid the images. 

But then there'll be no colors and font styles in the README. Plus asciicast supports animations, and there will a lot more of those once the progress bar support is merged.

**Comment by hadley** [comment]
Yeah, I was thinking you'd still use that in the README. And for animations, but you no longer have to use asciicast everywhere.

**Comment by gaborcsardi** [comment]
That's possible, with the caveat that it would look slightly different in the README and for animations. I'll take a look before the next release.

**Comment by gaborcsardi** [comment]
It would be considerable effort to match the themes, so they look similar, and also to refactor the article code, so that the animations are created in a background process, but the static output is not.

cli does not have any vignettes now, only articles, there are no images in the R package.

So the benefits are little, and the effort would be considerable, so if you don't mind, I'll not do this now.

**Comment by hadley** [comment]
Yeah, no rush. It's not that important.
